### PR TITLE
Add PRR soft freeze to 1.23 release schedule

### PR DESCRIPTION
**Pull request by ehashman**
#### What type of PR is this:

/kind cleanup

#### What this PR does / why we need it:

/cc @jeremyrickard @johnbelamaric @deads2k @wojtek-t 

prompted by https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ/

### CODE REVIEW

**Comment by reylejano** [comment]
Thank you @ehashman and the PRR team for all the time and effort conducting reviews.
I agree that a deadline for PRR questionnaires is needed, 1 week prior to the start of Enhancements Freeze is good

We can have the PRR questionnaire deadline on comms regarding the 1.23 release.

Informal LGTM for additional feedback from SIG Release or the 1.23 Enhancements lead, @salaxander

**Comment by salaxander** [comment]
/lgtm

I think this makes total sense. Ya'll were absolutely amazing last time getting all those PRRs done right in time for enhancements freeze, I would love for it to be a little more chill this time around :) I've added the soft freeze date to the dates section on the 1.23 tracking sheet as well.

**Comment by JamesLaverack** [comment]
/lgtm

I think, anecdotally, that the PRR was usually one of the last things to be done — so if this were a *hard* deadline then I suspect this would end up being just an earlier enhancements freeze. But as a soft deadline I'm pretty comfortable with it, especially with the logic of "you can ask after this deadline, but there is no guarantee the PRR team will get to it".

I know how busy it was for the PRR team last time around, so anything to make the days leading up to that deadline easier is a good idea.

**Comment by jrsapi** [comment]
This makes sense as the last few cycles we are chasing down if PRR approvals are in motion or not. This will make the enhancement freeze just a little less hectic.

File: releases/release-1.23/README.md [hunk 1]
```diff
@@ -54,6 +55,7 @@ The 1.23 release cycle is proposed as follows:
 | 1.23.0-alpha.1 released | Branch Manager | Wed August 25, 2021 | week 1 | |
 | Schedule finalized | Lead | Thu August 26, 2021 | week 1 | |
 | Team finalized | Lead | Fri August 27, 2021 | week 1 | |
+| Production Readiness Soft Freeze | Enhancements Lead | Thu September 2, 2021 | week 2 | |
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:58]: What’s the difference between a soft freeze and a regular freeze? 
```suggestion
| Production Readiness Freeze | Enhancements Lead | Thu September 2, 2021 | week 2 | |
```

File: releases/release-1.23/README.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ description: |
 The 1.23 release cycle is proposed as follows:
 
 - **Monday, August 23, 2021**: Week 1 - Release cycle begins
+- **Thursday, September 2, 2021**: Week 2 - [Production Readiness Soft Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
```
Comment (saschagrunert) [inline releases/release-1.23/README.md:41]: Let’s put the mail content into a official doc within this repository.

**Review by saschagrunert — COMMENTED** [review]
Generally LGTM, just two nits.

File: releases/release-1.23/README.md (hunk 2 above)
Comment (reylejano) [inline releases/release-1.23/README.md:41]:
```suggestion
- **Thursday, September 2, 2021**: Week 2 - [Production Readiness Freeze](https://groups.google.com/g/kubernetes-sig-architecture/c/a6_y81N49aQ)
```
I suggest not to differentiate between a soft freeze and a regular freeze. An exception to the PRR freeze can be part of an [exception request](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md)

Comment (johnbelamaric) [reply releases/release-1.23/README.md:41]: We do not want to require an exception request for folks being late with this. We don't need another hard deadline that requires exception requests - I think that would be overly burdensome.

**Comment by reylejano** [comment]
The 1.22 retro is on Thursday, Aug 19 at 10am PDT / 1pm EDT / 17:00 UTC, there are a few topics on the [1.22 retro agenda](https://docs.google.com/document/d/1deFXjK07TyGFds1wk2LjkRanqpjoGVmTFqmwTUgsSEM/edit) related to the PRRs.
It would be great if we can get feedback from the PRR team: @ehashman @johnbelamaric @wojtek-t @deads2k 

Changes can be done in a follow-up PR

/approve

**Pull request merged**